.travis.yml

```diff
@@ -24,17 +25,19 @@ install:
 - pip install tox-travis
 
 script: tox
-
+matrix:
+  allow_failures:
```

Comment:
Add `nightly` and allow it to fail?  Better to remove `nightly` in this case.

Reply:
But we need to know if there are some early changes which will break our code, but actually it is allowed to fail because it is a nightly version.

Reply:
No. When 3.9 becomes stable we fix everything (if there are anything to fix)